prow/plugins/lgtm/lgtm.go

```diff
@@ -367,7 +365,7 @@ func handlePullRequest(log *logrus.Entry, gc githubClient, config *plugins.Confi
 				}
 			}
 			// Get the current tree-hash
```

Comment:
We should check if `lastLgtmTreeHash` is non empty before we use an API token to get the commit. If it is empty we have no chance of matching the tree hash.